### ISSUE TITLE
Error: Uninterpreted extension 'bs.obj' with latest v1.0.0 and v1.0.1 releases

### ISSUE THREAD

**Issue by tmepple** [jaredly/reason-language-server#104]
With the latest 1.0.1 release I am now getting an error on my file which worked great using the previous release .vsix (beta-7) at commit 647e1139.

Basically I am getting `Error: Uninterpreted extension 'bs.obj'` at `($usernameOrEmail: String!, $password: String!, $remember: Boolean!)` in line 3 of the following snippet:

```reason
module CreateLoginMutation = [%graphql
  {|
  mutation login($usernameOrEmail: String!, $password: String!, $remember: Boolean!) {
    login(usernameOrEmail: $usernameOrEmail, password: $password, remember: $remember) {
      user { username email firstName lastName }
      errors { key message }
    }
  }
  |}
];
```

I'm not sure if there are other errors in this file as I think it stops after it finds the first error.

I tried `bsb -clean-world -make-world` to ensure everything was built but that hasn't helped.  Running `bsb` like this doesn't give any errors.  I've reverted to beta-7 at this point which works great.  Please let me know any questions to help troubleshoot.

[debug.log](https://github.com/jaredly/reason-language-server/files/2308240/debug.log)

**Comment by joealden** [comment]
Same here

**Comment by jsiebern** [comment]
And here

**Comment by jaredly** [comment]
hmmmm looks like the ppxs are being applied in the wrong order...

**Comment by jsiebern** [comment]
Probably related to #109 then? That is a ppx as well as far as I can recall, right?

**Comment by jaredly** [comment]
Ok, this happened when I tried to fix #89... looks like that one was a bucklescript bug 🙃 . I'll switch it back
